dbt_project.yml

```diff
@@ -1,11 +1,11 @@
 #Welcome to your Spellbook!
 
-name: 'spellbook'
-version: '1.0.0'
+name: "spellbook"
```

Comment:
just out of curiosity, were these intentionally changed to double quotes and have a specific reason? i'm fine either way, wondering if there was an issue

Reply:
I have dbt formatter extension installed on my VSCode, and it auto formats this to double quotes. I even tried to disable the formatter temporarily but somehow it still force formatted to double quotes whenever I saved 🫠, so I just left it be

Reply:
which one do you have? maybe we can all use it.

Reply:
may be worth it for us all to look at internally (although could still get external user PRs which don't use it)